src/BugReporting.jl

```diff
@@ -157,7 +166,18 @@ function upload_rr_trace(trace_directory)
     """)
     println("To upload a trace, please authenticate, by visiting:\n")
     println("\thttps://github.com/login/oauth/authorize?client_id=$GITHUB_APP_ID&state=$(HTTP.escapeuri(connectionId))")
-    s3creds = take!(c)
+    println("\n", "You can cancel upload by `Ctrl-C`.")
+    flush(stdout)
+    s3creds = try
+        take!(c)
+    catch err
+        if err isa InterruptException
```

Comment:
Unfortunately, it looks like `InterruptException` is not catchable _sometimes_. See also: https://github.com/JuliaLang/julia/issues/35713